src/main/java/Task.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * The Task class is a parent class for the different types of tasks that the user
+ * input will generate.
+ */
+class Task {
```

Comment:
Can consider defining as abstract class for extensibility purposes. Nevertheless, for now it is suitable.